documents/Libraries/pbrlib/gen-glsl/ogsfx/mx_lighting_functions.glsl

```diff
@@ -1,6 +1,6 @@
 // --------------------------------- OgsFx Lighting Functions --------------------------------------
 
-int getLightType(int index)
+int mx_getLightType(int index)
```

Comment:
When these function names are changed you also need to update the OGSFX_GET_LIGHT_DATA_MAP in OgsFxShaderGenerator.cpp, line 54.